docs/README.md

```diff
@@ -438,9 +438,9 @@ You can expand any of the messages that matches the filter to see the full stack
 
 Once you have finished exploring the Cloud Operations Sandbox project, don't forget to destroy it to avoid incurring additional billing.
 
-Destroy your Sandbox project by opening the Cloud Shell and running the destroy script:
+Destroy your Sandbox project by opening the Cloud Shell and running the destroy command:
 ```
-$ destroy.sh
+$ sandboxctl destroy
 ```
 
 This script destroys the current Cloud Operations Sandbox project. If the install.sh script were run again, a Cloud Operations Sandbox project with a new project id would be created.
```

Comment:
Nit for consistency since we stopped referring to "install.sh" to users, change "If the install.sh script were run again" to "If the `sandboxctl create` script were run again"

Reply:
Also you may want to update `/hack/scratchpad.sh` line 30-31 to use `sandboxctl` as well
Do we use the scripts inside of /hack actually?

Reply:
Good catch, I missed that one.

@simonz130 do you know if we still need `/hack`? It looks like it has old release scripts, but not the one we currently use